schemas/ihcl-gift-cards/index.ts

```diff
@@ -30,12 +30,18 @@ export const giftCards: FeatureSchemaDefinition = {
         value: "giftCards.card.order-status-form",
       },
     ],
-  
+
     tabs: [
       {
         title: "[Gift Card] Manage Card Multiple Tabs",
         value: "giftCards.tabs.manage-card-multiple-tabs",
       },
     ],
+    stepper: [
```

Comment:
please attach UI screen shots

Reply:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/120178389/230076630-615f6f49-bffa-46b2-9f68-ecfb98b73d6a.png">

Reply:
removed the var